packages/shared/lib/contexts/onboarding/actions/completeOnboardingProcess.ts

```diff
@@ -12,15 +12,18 @@ import { cleanupOnboarding } from './cleanupOnboarding'
 export async function completeOnboardingProcess(): Promise<void> {
     addOnboardingProfile()
 
-    /**
-     * NOTE: If recovering from a Stronghold that already contains
-     * at least one account, calling this function will cause
-     * an "AccountAliasAlreadyExists" error from wallet-rs. This
-     * error does NOT cause side-effects while creating a profile.
-     */
-    await createNewAccount()
+    const shouldCreateNewAccount = get(onboardingProfile)?.setupType !== ProfileSetupType.Claimed
```

Comment:
Depends which is merged first we might need to move this?

Reply:
Yeah this can be moved / adjusted. It's also okay if the loading screen PR gets merged first